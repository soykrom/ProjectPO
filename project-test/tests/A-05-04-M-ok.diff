*** tests/A-05-04-M-ok.out	2019-11-20 15:11:09.000000001 +0000
--- tests/A-05-04-M-ok.outhyp	2019-12-03 11:54:16.000000001 +0000
***************
*** 14,29 ****
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 2 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
! 5 - Chewbacca - chewbacca2@milenium.falcon.org - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
  4 - Darth Vader - darth2@dark.force.coAm - NORMAL - ACTIVO
! 7 - Han Solo - han.solo@milenium.falcon.org - NORMAL - ACTIVO
  9 - Han Solo - han.solo2@milenium.falcon.org - NORMAL - ACTIVO
! 0 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  3 - Luke Skywalker - last.jedi2@universe.com - NORMAL - ACTIVO
! 6 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  8 - Rey - real.last.jedi2@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
--- 14,29 ----
  4 - Mostrar notificações de utente
  5 - Saldar multa de utente
  0 - Sair
! Escolha uma opção: 5 - Chewbacca - chewbacca2@milenium.falcon.org - NORMAL - ACTIVO
! 2 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
  4 - Darth Vader - darth2@dark.force.coAm - NORMAL - ACTIVO
! 1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
  9 - Han Solo - han.solo2@milenium.falcon.org - NORMAL - ACTIVO
! 7 - Han Solo - han.solo@milenium.falcon.org - NORMAL - ACTIVO
  3 - Luke Skywalker - last.jedi2@universe.com - NORMAL - ACTIVO
! 0 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
  8 - Rey - real.last.jedi2@universe.com - NORMAL - ACTIVO
+ 6 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
  Menu de Gestão de Utentes
  1 - Registar utente
  2 - Mostrar utente
