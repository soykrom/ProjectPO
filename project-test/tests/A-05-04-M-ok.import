BOOK:A arte de sobreviver no 36:Joao Fonseca:30:FICTION:1234567892:22
BOOK:Analise Matematica sem Mestre:Carlos Fonseca:19:SCITECH:1234567890:3
DVD:Era uma vez na Amadora:Fernando Fonseca:20:FICTION:200505550:10
DVD:48 Horas para o Exame:Orlando Fonseca:12:FICTION:200505553:10
BOOK:Bairro Alto e o Budismo Zen:Zun Tse Fonseca:25:FICTION:1234567891:50
BOOK:Dicionário de Programação:Odete Fonseca:20:REFERENCE:1234567890:50
USER:Luke Skywalker:last.jedi@universe.com
USER:Darth Vader:darth@dark.force.coAm
USER:Chewbacca:chewbacca@milenium.falcon.org
USER:Luke Skywalker:last.jedi2@universe.com
USER:Darth Vader:darth2@dark.force.coAm
USER:Chewbacca:chewbacca2@milenium.falcon.org
USER:Rey:real.last.jedi@universe.com
USER:Han Solo:han.solo@milenium.falcon.org
USER:Rey:real.last.jedi2@universe.com
USER:Han Solo:han.solo2@milenium.falcon.org
DVD:Lumiar Selvagem:Pedro Fonseca:20:FICTION:200505551:5
DVD:The rise of Skywalker:J. J. Abrams:20:SCITECH:200505551:1
