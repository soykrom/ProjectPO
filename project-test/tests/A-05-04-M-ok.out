Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: 2 - Chewbacca - chewbacca@milenium.falcon.org - NORMAL - ACTIVO
5 - Chewbacca - chewbacca2@milenium.falcon.org - NORMAL - ACTIVO
1 - Darth Vader - darth@dark.force.coAm - NORMAL - ACTIVO
4 - Darth Vader - darth2@dark.force.coAm - NORMAL - ACTIVO
7 - Han Solo - han.solo@milenium.falcon.org - NORMAL - ACTIVO
9 - Han Solo - han.solo2@milenium.falcon.org - NORMAL - ACTIVO
0 - Luke Skywalker - last.jedi@universe.com - NORMAL - ACTIVO
3 - Luke Skywalker - last.jedi2@universe.com - NORMAL - ACTIVO
6 - Rey - real.last.jedi@universe.com - NORMAL - ACTIVO
8 - Rey - real.last.jedi2@universe.com - NORMAL - ACTIVO
Menu de Gestão de Utentes
1 - Registar utente
2 - Mostrar utente
3 - Listar utentes
4 - Mostrar notificações de utente
5 - Saldar multa de utente
0 - Sair
Escolha uma opção: Menu Principal
1 - Abrir dados
2 - Guardar dados
3 - Ver data
4 - Avançar data
5 - Menu de gestão de utentes
6 - Menu de gestão de obras
7 - Menu de gestão de requisições
0 - Sair
Escolha uma opção: 